example/example/contrib/sites/migrations/0004_auto_20201111_1444.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.1.3 on 2020-11-11 14:44
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sites', '0003_set_site_domain_and_name'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='site',
+            options={'ordering': ['domain'], 'verbose_name': 'site', 'verbose_name_plural': 'sites'},
+        ),
+    ]
```

Comment:
This seems to have snuck in this PR.  

This is b/c I use custom `MIGRATION_MODULES` to bypass the default migrations (or rather to add one of my own).  So apparently Django has gone and updated their sites framework since I first did that.

There's no harm in adding this file, especially as it's only used by the *example* project.  But it has nothing to do w/ the work in the current PR.

Reply:
I just assumed it was cause I added `uuid` to the model, I didn't even give it a second glance, will be more careful in future.